### ISSUE TITLE
Allow special slugs for collections ?

### ISSUE THREAD

**Issue by srosset81** [assemblee-virtuelle/activitypods#216]
To avoid having custom collections with simple hash (contrary to pod provider generated collections), we could allow to use a slug like `{objectPath}/attendees` ?

We would need to check the actor has write permission on the object to avoid fake collections naming ?

Right now the slug is sanitized, and the slashes are removed.

**Comment by Laurin-W** [comment]
I like the idea of supporting something human-readable :)

Do you mean, the URI should be completely custom or should it be something like `https://pod.example/ef839abd/attendees`?

**Comment by srosset81** [comment]
Yes it should be something like `https://pod.example/ef839abd/attendees`

**Comment by srosset81** [comment]
I don't think we will worry about this kind of "cosmetic" questions since with NextGraph it won't be possible anymore to have custom slug. 

The new data browser in Pods should make it easier for developer to navigate through data, no matter their URI !
